portfolio/src/main/webapp/script.js

```diff
@@ -30,3 +30,20 @@ function addRandomFunFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
 }
+
+var i = 1;
```

Comment:
"i" is often used as the variable in loop iterations. This seems to be a global variable, so I'd suggest to use a different and more descriptive name